### DISCUSSION TITLE
After advanced installation I get a bad gateway error

### DISCUSSION THREAD

**Discussion by albi** [teslamate-org/teslamate#1241] [Q&A]
I've installed Teslamate once before. It was running fine. Then I've uninstalled Docker and everything from Teslamate. After installing it again with the instructions "Advanced installation with Traefik, Let's Encrypt & HTTP Basic Auth" I get the bad gateway error, after entering my own generated credentials for the HTTP Basic Auth.

Don't know where to begin with debugging.

Alex

**Comment by albi** [comment]
Updated to the latest version. The Grafana Dashboard is available at https://teslamate.mydomain.de/grafana but not https://teslamate.mydomain.de/

**Reply by tonsument** [reply]
Were you able to resolve the issue? My Teslamate has been running fine (on Azure, Advance installation) until about a month ago. Since then I am getting Bad Gateway errors after entering credentials. After rebooting it will work fine for about a day, after which I am getting the Bad Gateway error again. Grafana is indeed running fine using the direct URL, but no (additional) data is collected. 
I also do not know where to start debugging.
